src/Mod/Part/Gui/Command.cpp

```diff
@@ -1330,7 +1379,7 @@ CmdPartExtrude::CmdPartExtrude()
 void CmdPartExtrude::activated(int iMsg)
 {
     Q_UNUSED(iMsg);
-    Gui::Control().showDialog(new PartGui::TaskExtrusion());
+    PartGui::checkSkipDialog(new PartGui::TaskExtrusion(), "makeExtrude", QT_TR_NOOP("Extrude"));

```

Comment:
Better explicitly use the FreeCAD dedicated context :
```suggestion
    PartGui::checkSkipDialog(new PartGui::TaskExtrusion(), "makeExtrude", QT_TRANSLATE_NOOP("Command", "Extrude"));
```
Same for others.

Reply:
Done.  Applied locally.